src/utils/stringUtils.ts

```diff
@@ -1,3 +1,7 @@
 export const capitalizeFoersteBokstav = (ord: string): string => {
   return ord.charAt(0).toUpperCase() + ord.slice(1);
 };
+
+export const firstLetterOnlyCapitalised = (word: string): string => {
+  return capitalizeFoersteBokstav(word.toLowerCase());
+};
```

Comment:
Er det ikke bedre å heller legge til en `ord.slice(1).toLowerCase()` i funskjonen ovenfor?

Reply:
Tja, man kan jo merge de to metodene, men jeg vet ikke om det er naturlig å anta at en metode som heter "gjør første bokstav stor", også gjør alle andre bokstaver små?

Vi kan evt. lage en lapp på å refaktorere? Denne metoden (eller en ny, refaktorert greie) må uansett tas i bruk de stedene hvor Anders gjorde `capitalizeFoersteBokstav(word.toLowerCase())` "manuelt".

Reply:
Kanskje en dristig antagelse, men i de alle fleste tilfeller vil man ha stor forbokstav, fulgt av små bokstaver?

Reply:
Synes det er litt overflødig å lage en lapp i Trello for å "fjerne" en metode som egentlig ikke trengs.

Metoden `firstLetterOnlyCapitalised`  trengs ikke dersom man enten bruker `capitalizeFoersteBokstav ` eller evt. gjør `capitalizeFoersteBokstav` skuddsikker med en legge til det som ble foreslått over.

Reply:
Jeg tror nok det stemmer, ja! Jeg tenker mest på at det burde reflekteres i metodenavnet hvis vi endrer resten av strengen også 🤔